### PR TITLE
Group 2: Attention Economy Finished Outline

### PR DESCRIPTION
**Pull request by lao9**

### CODE REVIEW

File: m4_sessions/1706-inning/group_two.md [hunk 1]
```diff
@@ -13,27 +17,35 @@ Students should be able to do the following:
...
-  * What could Google do as a company, or us as developers do to change the attention economy landscape? 
+  * Briefly define “Attention Economy”, as the current state of the scarce resource of “Attention” in a world of nearly infinite information.
+  * When people are asked if they think their time spent on most of these types of applications, over 90% regret how they spent their time. Present self (in the moment, using application) vs. Future Self (with feelings of regret)
+  * The reason these technologies are so successful:
+   * Take advantage of our psychology 
+   * Machine learning identifies what we like (even if it’s things that create outrage, actually especially outrage) 
+   * Tristian Harris - technology ethicist (inspiration of this gear up) took a class in “Persuasive Technology” during his time at Stanford
+   * Some of his classmates went on to create Instragram and LinkedIn
+   * However, there was never a discussion about if taking advantage of the human psyche in this manner is ethical
+   * Watch clips from Youtube: https://www.youtube.com/watch?v=50R21mblLb0
```
Comment (emhickmann) [inline m4_sessions/1706-inning/group_two.md:31]: Which clips are you going to watch?

Comment (emhickmann) [reply m4_sessions/1706-inning/group_two.md:31]: This is a great video!

File: m4_sessions/1706-inning/group_two.md [hunk 2]
```diff
@@ -13,27 +17,35 @@ Students should be able to do the following:
...
+  * When people are asked if they think their time spent on most of these types of applications, over 90% regret how they spent their time. Present self (in the moment, using application) vs. Future Self (with feelings of regret)
+  * The reason these technologies are so successful:
+   * Take advantage of our psychology 
+   * Machine learning identifies what we like (even if it’s things that create outrage, actually especially outrage) 
+   * Tristian Harris - technology ethicist (inspiration of this gear up) took a class in “Persuasive Technology” during his time at Stanford
+   * Some of his classmates went on to create Instragram and LinkedIn
+   * However, there was never a discussion about if taking advantage of the human psyche in this manner is ethical
+   * Watch clips from Youtube: https://www.youtube.com/watch?v=50R21mblLb0 
+
+* Small Group Discussion Time (15 mins)
```
Comment (emhickmann) [inline m4_sessions/1706-inning/group_two.md:33]: Are they picking which topic?  These are great questions!!!

File: m4_sessions/1706-inning/group_two.md [hunk 3]
```diff
@@ -13,27 +17,35 @@ Students should be able to do the following:
...
+  * What are some other examples of how you’re susceptible to the ‘attention economy’ that you’ve noticed?  [Dave]
+  * Are there any benefits to the “Attention Economy”? [Brett]
+  * Whose responsibility is it to spend our time wisely (technology companies or us as human)? [Craig]  
+  * If it’s up to the company, what are some strategies they could employ to ensure that their users are spending their time well while using their applications, while also maintaining revenue? 
+  * If the responsibility is us on as humans to avoid clicking and using, what could we do to spend our time wisely, while also not exhibiting too much “FOMO”? 
+
+* Whole Group Discussion (25 mins)
+ * Have each group present some of their findings (2 mins each)
```
Comment (emhickmann) [inline m4_sessions/1706-inning/group_two.md:41]: LOVE THIS!  Great direct example that is very complicated.

File: m4_sessions/1706-inning/group_two.md [hunk 4]
```diff
@@ -13,27 +17,35 @@ Students should be able to do the following:
...
+ * Let’s say you have a meditation application that you know will help people, but the only way compete with Facebook, Snapchat, etc. is to add daily notifications and maybe a meditation streak game, which kind of defeats the purpose of an app that’s supposed to bring you emotional peace. 
+   * As a junior, what do you do? 
+   * As a more senior “technologist” (senior dev, or founder / CTO)
+
```
Comment (emhickmann) [inline m4_sessions/1706-inning/group_two.md:45]: I would add some sort of "closing" even if it is final thoughts from each of you sharing your opinions/answer to the question.

File: m4_sessions/1706-inning/group_two.md [hunk 5]
```diff
@@ -13,27 +17,35 @@ Students should be able to do the following:
...
 * Provide some other resources for them to explore further:
   * https://www.samharris.org/podcast/item/what-is-technology-doing-to-us 
+  * http://www.cbsnews.com/videos/brain-hacking-2/
  
 ### Names of group members: Dave Hubertis, Craig Ness, Mark Stover, Brett Schwartz, Lauren Oliveri
```
Comment (emhickmann) [inline m4_sessions/1706-inning/group_two.md:50]: @lao9 Your group is in a great place.  I want to go to this session!!

**Review by emhickmann — COMMENTED** [review]
@lao9 Awesome job with this session.  Check these comments as you do final preparation.

File: m4_sessions/1706-inning/group_two.md (hunk 5 above)
Comment (lao9) [reply m4_sessions/1706-inning/group_two.md:50]: Thanks for the feedback @emhickmann !! I'll pass these along to the team.

File: m4_sessions/1706-inning/group_two.md (hunk 1 above)
Comment (lao9) [reply m4_sessions/1706-inning/group_two.md:31]: Oops - we're going to watch just that youtube video - it's only 10 minutes! Originally we were going to pick out clips from a 60 minutes interview, but this video is more concise, while also being more comprehensive! It's based on the podcast that inspired this Gear Up.

**Pull request merged**